### PR TITLE
Java 1.8

### PR DESCRIPTION
**Pull request by sreedishps**

### CODE REVIEW

**Comment by rajubairishetti** [comment]
+1

Users can not run on JRE7 with this new package. Any features/changes which will be added in pintail will be available to only users who is running on JRE8 or higher versions.
@sreedishps, could you please check with @zulfi.

**Comment by zulfiali** [comment]
Sreedish,

Can we parameterise these changes ? In a way that we can specify source and target version in mvn command while building the package ?

**Comment by sreedishps** [comment]
@zulfiali  yes we can parameterise the JDK version. I will add a property in the pom,(which will default the JDK version to 1.8), and overrides can be provided via command-line,and the command-line takes precedence. @rajubairishetti Please mark your view on this also

**Comment by sreedishps** [comment]
MAde JDK parameterised

File: pom.xml [hunk 1]
```diff
@@ -229,8 +230,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>${jdk.version}</source>
+          <target>${jdk.version}</target>
```
Comment (rajubairishetti) [inline pom.xml]: Can you change property  to *java.version* as mentioning jdk does not seem to be good for target?

Comment (sreedishps) [inline pom.xml]: Done

**Comment by rajubairishetti** [comment]
+1

**Pull request merged**